patches/httpOnly.patch

```diff
@@ -13,36 +13,30 @@
      my $charset = &get_charset();
      &PrintHeader($charset);
 
---- authentic-theme/pam_login.cgi  2019-10-01 18:22:33.051430189 +0000
-+++ authentic-theme/pam_login.cgi  2019-10-01 18:34:29.781336839 +0000
-@@ -64,11 +64,12 @@
+--- authentic-theme/pam_login.cgi	2021-04-29 17:05:26.000000000 -0400
++++ authentic-theme/pam_login.cgi	2021-04-29 17:10:53.000000000 -0400
+@@ -63,9 +63,7 @@
  }
 
  my $sec = lc(get_env('https')) eq 'on' ? "; secure" : "";
+-if (!$miniserv{'no_httponly'}) {
+-    $sec .= "; httpOnly";
+-}
 +$sec .= "; httpOnly";
  my $sidname = $miniserv{'sidname'} || "sid";
  print "Auth-type: auth-required=1\r\n";
  print "Set-Cookie: banner=0; path=/$sec\r\n"   if ($gconfig{'loginbanner'});
- print "Set-Cookie: $sidname=x; path=/$sec\r\n" if ($in{'logout'});
--print "Set-Cookie: redirect=1; path=/\r\n";
-+print "Set-Cookie: redirect=1; path=/$sec\r\n";
```

Comment:
I think this might still be needed, no?
We added this to prevent OpenVAS from raising a vulnerability against ourselves for not having a secure cookie.

Reply:
Have something secure is always better. 
Actually it was in previous patch as well. I've just updated diff with new offset

Reply:
So long as the final output has the `httpOnly` value, I'm happy.